### PR TITLE
Fix/update README reflecting the code re-factor of martha_v3 [No Jira]

### PR DESCRIPTION
**Pull request by rsasch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,7 +164,7 @@ It will always return an object with the same properties:
  contentType:    string,
  size:           int,
  timeCreated:    string [dos objects only],
- timeUpdated:        string [usually],
+ updated:        string [usually],
```
Comment (aednichols) [inline README.md]: I know this is a pre-existing condition, but what does "usually" mean?

**Review by aednichols — APPROVED** [review]

**Review by salonishah11 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kshakir) [reply README.md]: Can come back as `null` also. I'm not an expert in JSON, but I believe than null is a different type than a string.

Comment (rsasch) [reply README.md]: To put a finer point on it, Martha is at the mercy of whatever it gets back from the URI resolver, so there's no guarantee that it won't be `null`.

Comment (aednichols) [reply README.md]: Cool. This is very much a seagull/out of scope request but can we say
```suggestion
 updated:        string [string by design, resolver sometimes returns null]
```
to capture this conversation?

**Pull request merged**